cmd/registry/controller/test/manifest.yaml

```diff
@@ -12,32 +12,32 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-project: "controller-demo"
-manifest:
-- resource: apis/-/versions/-/specs/-/artifacts/lint-gnostic
+name: "test-manifest"
```

Comment:
If we're changing the structure of the manifest, should the other test files change too? Or what's the difference between those manifests with "project" fields and these ones with "name" fields?

Reply:
Yes, the other manifests should also change, the controller tests are currently dependent on those manifests. I am going to cleanup and update those as part of the `controller` command cleanup. I wanted to keep this change short and not include too many cleanup changes.